test/services/supported_ssg_updater_test.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# A class to test the SupportedSsgUpdater service
+class SupportedSsgUpdaterTest < ActiveSupport::TestCase
+  context 'fallback file' do
+    setup do
+      @supported_file = ::Tempfile.new('suported_ssg_file')
+      @supported_file.write(File.read(SupportedSsgUpdater::SUPPORTED_SSG_FILE_PATH))
+      @supported_file.rewind
+
+      @fallback_file = ::Tempfile.new('fallback_ssg_file')
+      @fallback_file.write(File.read(SupportedSsgUpdater::DEFAULT_SUPPORTED_SSG_FILE_PATH))
+      @fallback_file.rewind
+
+      SupportedSsgUpdater.stub_const(:SUPPORTED_SSG_FILE_PATH, @supported_file.path) do
+        SupportedSsgUpdater.stub_const(:DEFAULT_SUPPORTED_SSG_FILE_PATH, @fallback_file.path) do
+          SupportedSsgUpdater.update_ssg_fallback_yaml
+        end
+      end
```

Comment:
I'd rather move this into each test, as it's not really part of the setup :thinking: